src/DynamoCoreWpf/Views/FileTrust/FileTrustWarning.xaml.cs

```diff
@@ -1,15 +1,20 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Windows;
 using System.Windows.Controls;
 using System.Windows.Controls.Primitives;
+using System.Windows.Media;
 using System.Windows.Threading;
 using Dynamo.Controls;
 using Dynamo.Models;
+using Dynamo.Utilities;
 using Dynamo.ViewModels;
-using Dynamo.Wpf.Controls;
+using Dynamo.Wpf.UI.GuidedTour;
+using Dynamo.Wpf.Utilities;
 using Dynamo.Wpf.ViewModels;
 using Dynamo.Wpf.ViewModels.FileTrust;
+using Res = Dynamo.Wpf.Properties.Resources;
```

Comment:
Let's stick to regular namespace naming guidelines `using Dynamo.Wpf.Properties.Resources`
https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards#namespaces

Reply:
I removed this namespace and now is getting the resource name directly, see the changes in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12940/commits/580f37c1461e6f87e6305fe16271d169fb8bcb45